app/Http/Controllers/ExhibitionsController.php

```diff
@@ -87,8 +87,9 @@ protected function show($id, $slug = null)
             ->findOrFail((int) $id);
 
         // If the exhibition has not started or not ended, check if its augmented model is published before showing
```

Comment:
The logic here is intended to be "if the exhibition closes in the future and has not been intentionally published." For permanent exhibitions, there may be no end date entered at all, so we can only rely on the start date. We check both fields here as a catch all for bad data, in case only one of either field is filled in.